src/main/kotlin/no/nav/syfo/motebehov/api/internad/v2/MotebehovVeilederADControllerV2.kt

```diff
@@ -25,16 +28,25 @@ class MotebehovVeilederADControllerV2 @Inject constructor(
     private val metric: Metric,
     private val historikkService: HistorikkService,
     private val motebehovService: MotebehovService,
+    private val pdlConsumer: PdlConsumer,
     private val veilederTilgangConsumer: VeilederTilgangConsumer
 ) {
     @GetMapping(value = ["/motebehov"], produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentMotebehovListe(
         @RequestParam(name = "fnr") sykmeldtFnr: @Pattern(regexp = "^[0-9]{11}$") String
-    ): List<Motebehov> {
+    ): List<MotebehovVeilederDTO> {
         metric.tellEndepunktKall("veileder_hent_motebehov")
         val fnr = Fodselsnummer(sykmeldtFnr)
         kastExceptionHvisIkkeTilgang(fnr)
-        return motebehovService.hentMotebehovListe(Fodselsnummer(fnr.value))
+        val motebehovVeilederDTOList = motebehovService.hentMotebehovListe(Fodselsnummer(fnr.value))
+            .toMotebehovVeilederDTOList()
```

Comment:
Kan kanskje unngå toMotebehovVeilederDTOList ved å gå direkte på motebehovService.hentMotebehovListe(Fodselsnummer(fnr.value)).map { ... }

Reply:
Det var bevisst å ha en exension function for dette. Har begynt med å ha en slik extension function i API'ene hos ISYFO og synes det blir "cleanere" og mer gjenbrukbart.
Men om dere ikke vil ha det slik så endrer jeg 👍

Reply:
Jeg liker aproach som Erik har foreslått 👍🏻